### PR TITLE
fix(verify): Allow url asset not specifying path #382

### PR DESCRIPTION
**Pull request by awcjack**
Fix https://github.com/semantic-release/gitlab/issues/382

Validator accept url asset not specifying `path` property

### CODE REVIEW

File: lib/verify.js [hunk 1]
```diff
@@ -15,7 +15,7 @@ const canBeDisabled = (validator) => (value) => value === false || validator(val
 
 const VALIDATORS = {
   assets: isArrayOf(
-    (asset) => isStringOrStringArray(asset) || (isPlainObject(asset) && isStringOrStringArray(asset.path))
+    (asset) => isStringOrStringArray(asset) || (isPlainObject(asset) && isNonEmptyString(asset.url)) || (isPlainObject(asset) && isStringOrStringArray(asset.path))
```
Comment (fgreinacher) [inline lib/verify.js:18]: What about adding some brackets for readability and getting rid of the double `isPlainObject` call?

```suggestion
    (asset) => isStringOrStringArray(asset) || (isPlainObject(asset) && (isNonEmptyString(asset.url) || isStringOrStringArray(asset.path)))
```

**Review by fgreinacher — CHANGES_REQUESTED** [review]
Thanks @awcjack! 

I left a suggestion and we have some [lint issues](https://github.com/semantic-release/gitlab/runs/6644767911?check_suite_focus=true) 😄

File: lib/verify.js (hunk 1 above)
Comment (awcjack) [reply lib/verify.js]: Done

**Review by fgreinacher — APPROVED** [review]

**Comment by fgreinacher** [comment]
Thanks for taking care of this!

**Pull request merged**